pkg/manifests/manifests_test.go

```diff
@@ -1188,6 +1164,86 @@ func TestPrometheusK8sRemoteWriteURLs(t *testing.T) {
 	}
 }
 
+func TestRemoteWriteAuthorizationConfig(t *testing.T) {
+	for _, tc := range []struct {
+		name    string
+		config  string
+		checkFn func(*testing.T, monv1.RemoteWriteSpec)
```

Comment:
Since there can be many remote write targets, lets have a slice of checkFn(). The surrounding test should fail if len(checkFn) != len(p.Spec.RemoteWrite).

```suggestion
		checkFn []func(*testing.T, monv1.RemoteWriteSpec)
```

Reply:
Good idea!